### ISSUE TITLE
Griddly rollout example

### ISSUE THREAD

**Issue by aioun** [Bam4d/Griddly#117]
Hello,
I tested most of Griddly functions and I have trained the clusters (10x10) example using GAAgent, from griddly.util.rllib.torch. How ever, I was not able to  rollout using the checkpoint. Here is my error:
RuntimeError: Given groups=1, weight of size [32, 10, 3, 3], expected input[1, 312, 240, 3] to have 10 channels, but got 312 channels instead
Do you have and example of code where you are able to use the checkpoint for agent.compute_action(state).

Thanks

**Comment by Bam4d** [comment]
I dont have any specific examples for this.

From the size of your input vector and the trained weights, it looks like you might be training using `VECTOR` observers and then rolling out in the model with either `BLOCK_2D` or `SPRITE_2D` observers.

make sure you set `player_observer_type=gd.ObserverType.VECTOR` in the environment where you are rolling out.

**Comment by aioun** [comment]
Thank you ! I made it working now. As you have mentioned in your document "By default the agent sees a VECTOR view of the environment. This view is passed to a Global Average Pooling Agent to produce the policy."
In fact, there still 2 points. Obs is of type unit8 so I changed it to obs = obs.astype('float64')
In addition, in agent code you are using: permute(0, 3, 1, 2), so I changed my input to obs = np.moveaxis(obs, 0, 2).
Is this make sense to you ?
Best
